test/jdk/java/foreign/TestNoForeignUnsafeOverride.java

```diff
@@ -21,23 +21,14 @@
  * questions.
  */
 
-/*
- * @test
- * @run testng TestNoForeignUnsafeOverride
- */
-
-import jdk.incubator.foreign.MemoryAddress;
-
-import jdk.incubator.foreign.MemorySegment;
-import org.testng.annotations.Test;
+package org.openjdk.foreigntest;
 
-public class TestNoForeignUnsafeOverride {
-    static {
-        System.setProperty("foreign.restricted", "permit");
-    }
+import jdk.incubator.foreign.*;
 
-    @Test(expectedExceptions = IllegalAccessError.class)
-    public void testUnsafeAccess() {
-        MemorySegment.ofNativeRestricted();
-    }
+public class PanamaMain {
+   public static void main(String[] args) {
+       System.out.println("Trying to get CLinker");
+       CLinker.getInstance();
+       System.out.println("Got CLinker");
+   }
 }
```

Comment:
This file doesn't seem used? You already have the PanamaMain in org.openjdk.foreigntest.PanamaMain as well